sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamCalcRel.java

```diff
@@ -315,7 +317,7 @@ private static Expression castOutput(Expression value, FieldType toType) {
   private static Expression castOutputTime(Expression value, FieldType toType) {
     Expression valueDateTime = value;
 
-    // First, convert to millis (except for DATE type)
+    // First, convert to millis (except for DATE/TIME type)
```

Comment:
Seems we can combine the first and second step now (and update the comments). The code will look much simpler that way.

Reply:
Done. Thanks.